test/membrane_opus/parser_test.exs

```diff
@@ -92,7 +92,13 @@ defmodule Membrane.Opus.Parser.ParserTest do
   test "non-self-delimiting input and output" do
     inputs =
       @fixtures
-      |> Enum.map(fn fixture -> fixture.normal end)
+      |> Enum.map(fn fixture ->
+        %Buffer{
+          pts: fixture.pts,
+          payload: fixture.normal,
+          metadata: %{duration: fixture.duration}
```

Comment:
Is `metadata: %{duration: fixture.duration}` needed here and in the other places, where you create input for parser?